docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>Mention it will fill both <code>NA</code> and <code>NaN</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Modified this. </p>